erpnext_com/www/docs/user/manual/en/setting-up/email/document-follow.md

```diff
@@ -0,0 +1,36 @@
+<!-- add-breadcrumbs -->
+# Document Follow
+
+> Introduced in v12
+
+**Document Follow is feature that allows users to track changes in documents via email notifications.**
```

Comment:
Document Follow is a feature that allows users to track changes in documents via email notifications.

Reply:
done